consensus/bor/bor.go

```diff
@@ -454,6 +454,30 @@ func (c *Bor) verifyCascadingFields(chain consensus.ChainHeaderReader, header *t
 		return err
 	}
 
+	// Verify the validator list match the local contract
+	if IsSprintStart(number+1, c.config.CalculateSprint(number)) {
+		newValidators, err := c.spanner.GetCurrentValidators(context.Background(), header.ParentHash, number+1)
+		if err != nil {
+			return errUnknownValidators
+		}
+
+		sort.Sort(valset.ValidatorsByAddress(newValidators))
+
+		headerVals, _ := valset.ParseValidators(header.Extra[extraVanity : len(header.Extra)-extraSeal])
```

Comment:
What is the second param here? Is it an error? Could we not ignore it?

Reply:
Line 123?

Reply:
@cffls

Reply:
Ah you are right. Fixed it. Thanks!